ros2_modulo_control/Dockerfile

```diff
@@ -1,13 +1,17 @@
 ARG BASE_IMAGE=ghcr.io/aica-technology/ros2-modulo
 ARG BASE_TAG=galactic
 
-FROM ${BASE_IMAGE}:${BASE_TAG}
+FROM ${BASE_IMAGE}:galactic as ros2-control-version-galactic
 USER ${USER}
-ARG ROS2_CONTROL_TAG=1.0.0
 
```

Comment:
All of these lines can be removed now I think. The layer
```dockerfile
FROM ${BASE_IMAGE}:galactic as ros2-control-version-galactic
```
is declared correctly later so isn't needed here. The ARGs for BASE_IMAGE and BASE_TAG are also not consumed in that initial stage so can be removed.